### ISSUE TITLE
Support full markdown support for popovers

### ISSUE THREAD

**Issue by damithc** [MarkBind/markbind#53]
Current situation: When this is included in another page, the popover does not show properly. The content of the popover appears below the link. Line breaks seems to be the cause of the problem.
```html
<div id="xp">
<popover id="pop:xp" title="eXtreme Programming (XP)">eXtreme Programming (XP)
<div slot="content">

_Extreme programming (XP)_ is a software development methodology which is intended to improve software quality and responsiveness to changing customer requirements. As a type of agile software development, it advocates frequent "releases" in short development cycles, which is intended to improve productivity and introduce checkpoints at which new customer requirements can be adopted.  [wikipedia, 2017.05.01]

</div>
</popover>
</div>
```

![image](https://cloud.githubusercontent.com/assets/1673303/25575394/8f33bfd2-2e89-11e7-9b38-d6af0fd401b5.png)

**Comment by Gisonrg** [comment]
Popover already supports markdown rendering. The problem is related to double rendering (render when include + popover component rendering). Change to the following format should work:
```
<div id="xp">
<popover id="pop:xp" title="eXtreme Programming (XP)">eXtreme Programming (XP)
<span slot="content">
_Extreme programming (XP)_ is a software development methodology which is intended to improve software quality and responsiveness to changing customer requirements. As a type of agile software development, it advocates frequent "releases" in short development cycles, which is intended to improve productivity and introduce checkpoints at which new customer requirements can be adopted.  [wikipedia, 2017.05.01]
</span>
</popover>
</div>
```
Remove the line break to disallow rendering at `include` stage. Make it a `span` element to allow correct rendering in popover component.

**Comment by damithc** [comment]
> Change to the following format should work

When I do that, the content is correctly included in the popover, but the markdown syntax is not recognized. Here's the specific example from the site demo.

![image](https://cloud.githubusercontent.com/assets/1673303/25880015/62024d28-3567-11e7-9e22-161b156513aa.png)

**Comment by damithc** [comment]
A possible solution: support a `<md></md>` tag that can be used to mark a segment as 'definitively markdown' when the parser/renderer is not able to guess the markdown text correctly.

**Comment by acjh** [comment]
Resolved by #137
